### DISCUSSION TITLE
[log explore] loki: Error prompt optimization of logql query box of grafana loki explore

### DISCUSSION THREAD

**Discussion by liguozhong** [grafana/grafana#52358] [Observability]
**Is your feature request related to a problem? Please describe.**

When the logql input is wrong, I expect to be able to directly locate the wrong position in the ui console of grafana

![image](https://user-images.githubusercontent.com/9583245/177953096-526e7cbc-1a08-468c-8a25-97446c73b9e8.png)

**Describe the solution you'd like**
for example，that would be great
![image](https://user-images.githubusercontent.com/9583245/177953599-9112b79a-74c5-43c4-a0b3-1797478ab706.png)

**Describe alternatives you've considered**

**Additional context**

**Comment by liguozhong** [comment]
thanks @dannykopping

**Comment by liguozhong** [comment]
help wanted

**Reply by svennergr** [reply]
thanks @liguozhong. That looks like a good idea! I will raise this topic internally.

**Reply by liguozhong** [reply]
thanks ! great,It's awesome .
because we migrated from log-system with a search box like this to grafana loki , and our original users complained about losing this feature.

**Comment by liguozhong** [comment]
help wanted

**Comment by svennergr** [comment]
Hi @liguozhong,

sorry for not following up here. We added this feature request and will release it with Grafana 9.4.0.

Please find the PR for reference here: https://github.com/grafana/grafana/pull/59427

**Reply by liguozhong** [reply]
thanks 👍👍👍👍

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
